Composer/packages/client/src/utils/navigation.ts

```diff
@@ -108,7 +110,41 @@ export function convertDialogDiagnosticToUrl(diagnostic: Diagnostic): string {
   return uri;
 }
 
+export function convertInlineLgDiagnosticToUrl(targetTemplateId, dialogs): string {
+  let url = '';
+  function visitor(path: string, value: any): boolean {
+    if (has(value, 'activity') && value.activity === `@{${targetTemplateId}()}`) {
+      url = path;
+      return true;
+    }
+    return false;
+  }
+  dialogs.forEach(dialog => {
+    const triggers = get(dialog, 'content.triggers', []);
+
+    triggers.forEach((t, index) => {
+      searchLgTemplate(`dialogs/${dialog.id}?selected=triggers[${index}]&focused=triggers[${index}]`, t, visitor);
```

Comment:
if the logic here is `if find a target url, break loop`, you can add url as an judgment.
```
if (url) return url;
```
or you can extend `searchLgTemplate`' s return value as a judgment.